source/common_fields.rst

```diff
@@ -0,0 +1,484 @@
+
+In this field, you can manually enter a user identification number. This field is not updated by the inventory.
+
+.. image:: ../image/alternate-username-number.png
+   :alt: module assets - alternate username number
+   :align: center
+   :scale: 51%
+
+
+Comments
+~~~~~~~~
+
+It's just a text field in which you can indicate what you want.
+
+Computer type
+~~~~~~~~~~~~~
+
+The type of computer allows you to specify whether the machine is a laptop, a server, a workstation, etc.
+There is no predefined type; the different types must be created.
+
+.. image:: ../image/computer-type.png
+   :alt: module assets - computer-type
+   :align: center
+   :scale: 50%
+
+- To create a **computer type**, click **+**
+- Enter a **Name**
+- Enter a comment if it's necessary
+- Click **+ Add**
+
+The computer will be available for all entities. You can use **templates** or **rules** with computer type
+
+.. image:: ../image/add-computer-type.png
+   :alt: module assets - computer-type
+   :align: center
+   :scale: 64%
+
+Group
+~~~~~
+You can add a group manually or create a group directly by this field.
+
+
+.. image:: ../image/group.png
+   :alt: module assets - user
+   :align: center
+   :scale: 50%
+
+- To create a group, click on **+ Add**
+- If it's necessary, check the box **Child entities** (this group will be visible in the sub-entity of the one you are in)
+- Enter a **Name**
+- A **Code** (optionnal)
+- Choose if this group is **Child of**
+- Select yes or no to **Recursive membership** (if enabled, members of this group will also become implicit members of its children groups)
+- You can define whether this group can be **visible in a ticket** (as a requester, observer, assigned to, task, can be notified)
+- Define if the group can be a **manager in a project**
+- Select if this group **can contain** **Items** and/or **Users**
+- Click **+ Add**
+
+.. image:: ../image/add-group-1.png
+   :alt: module assets - group
+   :align: center
+   :scale: 63%
+.. image:: ../image/add-group-2.png
+   :alt: module assets - group
+   :align: center
+   :scale: 63%
+
+- In this window, you need to add users. In **Users** tab, select those you wish to associate with the group (prefer users with a technical or higher profile)
+
+.. image:: ../image/add-users-in-group.png
+   :alt: module assets - add user in group
+   :align: center
+   :scale: 43%
+
+1. Select **user**
+2. Define it this user can be **Manage** group (add/modify memebers, name, etc.)
+3. whether or not to authorise **Delegatee** rights (can open ticket for the group)
+4. **Add** your group
+
+Once the group has been created, you can add another or select it from the drop-down list.
+
+Group in charge
+~~~~~~~~~~~~~~~
+
+A group in charge can modify the information of the computer and his elements (status, computer type, etc.)
+
+.. image:: ../image/group-in-charge.png
+   :alt: module assets - group in charge
+   :align: center
+   :scale: 51%
+
+- To create a group, click on **+ Add**

```

Comment:
I've added a comment on the difference between these 2 groups. Even if the set-up procedures are identical, I think it's better to be as explicit as possible. When I talk about groups (in configuration) I'm going to have to refer to the procedure for adding a group to which the group in charge doesn't intervene. To avoid confusion, I think it's better to keep these 2 fields separate. I don't know if I'm making myself clear

Reply:
Yeah, I think I understand, seems OK.